packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -787,7 +792,7 @@ abstract class Element implements BuildContext {
     _parent = parent;
     _slot = newSlot;
     _depth = _parent != null ? _parent.depth + 1 : 1;
-    _active = true;
+    _owner = parent?._owner;
```

Comment:
I think the root node should be special-cased by having it manually set the owner. See comments above about owner in Element.